source/directives/uiTreeNode.js

```diff
@@ -68,11 +69,23 @@
               }
             });
 
-			scope.$watch('expandOnHover', function (val) {
+            scope.$watch('expandOnHover', function (val) {
               UiTreeHelper.setNodeAttribute(scope, 'expandOnHover', val);
               attrs.$set('expandOnHover', val);
             });
 
+            scope.$watch(attrs.appendTo, function(val) {
```

Comment:
I was following the earlier style, previous attribute watchers use the scope.$watch method (see lines 55 and 66).